model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -575,13 +575,27 @@ message StandardCoders {
     // Components: Coder for a single element.
     ITERABLE = 3 [(beam_urn) = "beam:coder:iterable:v1"];
 
-    // Components: None
-    TIMESTAMP = 4 [(beam_urn) = "beam:coder:timestamp:v1"];
+    // Encodes a timer containing a timestamp and a user specified payload.
+    // The encoding is represented as: timestamp payload
+    //   timestamp - an 8 byte integer shifted by -9223372036854775808 encoded
```

Comment:
Nit: Can you clarify the language with "shifted by"?  At first, I thought it means that to encode any integer, you add `-9223372036854775808`, but that isn't the case.

Reply:
you subtract that amount, updated the comment.